tests/test_studio.py

```diff
@@ -77,7 +79,9 @@ def test_post_to_studio(tmp_dir, mocker, monkeypatch):
             "step": 1,
             "metrics": {live.metrics_file: {"data": {"step": 1, "foo": 2}}},
             "params": {live.params_file: {"fooparam": 1}},
-            "plots": {scalar_path: {"data": [{"step": 1, "foo": 2.0}]}},
+            "plots": {
+                f"{live.dvc_file}::{scalar_path}": {"data": [{"step": 1, "foo": 2.0}]}
```

Comment:
It looks like this ended breaking Live plot updates in Studio. I am trying to isolate and understand why it does work for some plots but not others